Dfe.PrepareTransfers.DocumentGeneration.Tests/Helpers/HtmlToDocumentTests.cs

```diff
@@ -399,6 +399,11 @@ public void AddHeading(Action<IHeadingBuilder> action)
             throw new NotImplementedException();
         }
 
+        public void AddTextHeading(string headingText, HeadingLevel headingLevel)
+        {
+            throw new NotImplementedException();
```

Comment:
Can we either implement this or remove it if it isn't needed

Reply:
i think its required due to the interface that it inherits .. sonar cube was insiting it needs to be there

Reply:
Remove it from the interface?